webapp/channels/src/components/youtube_video/youtube_video.tsx

```diff
@@ -20,24 +21,126 @@ type Props = {
 
 type State = {
     playing: boolean;
+    useMaxResThumbnail: boolean;
+    prevLink: string;
+}
+
+type YouTubeThumbnailProps = {
+    play: () => void;
+    videoTitle: string;
+    onError: () => void;
+    thumbnailUrl: string;
+    useMaxResThumbnail: boolean;
+};
+
+function YouTubePrefix() {
```

Comment:
Could you remove the translation from `YouTubePrefix` and the corresponding string from the translation file?

Reply:
Makes total sense. Updated